_posts/release/2020-02-29-accumulo-1.10.0.md

```diff
@@ -0,0 +1,216 @@
++ AccGcWalErrors- number of errors during WAL deletion  
++ AccGcPostOpDuration - duration of compact / flush 
++ AccGcRunCycleCount - 1-up cycle count 
+
+#### Add option allow clone table to keep the clone offline.
++ Option to leave cloned tables offline (#1475) 
++ Updated the shell clone command with a -o option
+  
+#### Improved metadata table consistency checking and testing during GC
+    
++ During GC scans, an error will be thrown if the GC fails consistency checks (#1379)
++ Added a check to ensure the last tablet was seen.  This should cause an error to be thrown if 
+the scanner stops returning data for any reason.  
+
+### Java Language Level Changes
+This lists some of the code changes made to update the Java language level to Java 8.  They are
+listed here to aid migration of client code that may have similar issues migrating to the updated
+language level (#1467).
++ Update to maven compiler source and target to Java 1.8
++ Add optional dependency to pom for modernizer annotations
++ Add SuppressModernizer annotation to uses of Guava Predicate in AccumuloConfiguration and its 
+subclasses, Optional in VolumeChooser, and Function in GroupBalancer
++ Replace Guava Function, Predicate, Preconditions, Optional and Iterators where able with java.util equivalents
++ Replace core.util.Base64 class with java.util equivalent
++ Fix warnings introduced by Java 1.8
++ Drop method in MockConfiguration causing compiler warning and made PropertyFilter in 
+AccumuloConfiguration extend guava Predicate
++ Rename execute methods in MasterClient and ServerClient, like the 2.0 changes to fix the 
+overload warnings
++ Add colVis file to ContinuousInputFormat. Fixes #1368 (#1369)
++ Fix unused var and generics warning
++ Limit UnsynchronizedBuffer maximum size to Integer.MAX_VALUE - 8 (#1523)
++ Remove need for ANT on classpath (#1532)
++ Update spotbugs and maven enforcer plugin versions
++ Rename yield method - future restricted identifier
++ Fix javadoc html header elements for newer JDKs
+
+### Other Changes
+
+#### Build Improvements / Changes
++ Switch to GitHub Actions and improve CI workflow (#1671)
++ Switch POM project xml attrs from HTTP to HTTPS
++ Add patch file for Java 11 generated apidocs
++ Update native map makefile to work with Java 11.
++ The makefile was modified to check if javac exists - previously it checked to see if javah exists
+inorder to determine if java is a JDK or JRE.  In Java 11, javah no longer exists - changed to 
+look for javac compiler command that exists in Java 8 and 11.
+
+#### Test Improvements
++ Make ITs override defaultTimeoutSeconds (#1483)
++ Increase timeouts on some slow ITs (#1482)
++ Stabilize ChaoticBalancerIT (#1491)
++ Increase timeout for GarbageCollectWALIT to allow recovery
++ Improve importDirectory tests from \#1299
++ Limit log size in Travis CI builds
+
+# Property Change Summary
+
+Set server count threshold and maximum wait time on start-up before master begins tablet assignments:
```

Comment:
Are all of these new properties?

Reply:
Yes

Reply:
I compared 1.9.3 to the 1.9 branch and these are all new